build.gradle

```diff
@@ -1,14 +1,14 @@
 plugins {
     id 'application'
     id "org.sonarqube" version "3.0"
-    id 'org.springframework.boot' version '2.2.6.RELEASE'
+    id 'org.springframework.boot' version '2.3.0.RELEASE'
```

Comment:
You need to upgrade to the latest 2.3.x version, 2.3.6 seems to be the latest one, so please update the version.